modules/assim.sequential/R/SDA_NA_downscale.R

```diff
@@ -0,0 +1,459 @@
+#' @description
```

Comment:
filename includes "NA" which is confusing because 1. for most R users that means Not Available and 2. there's nothing North America specific to any of these functions. The NA specific bit is in the inst/anchor script

Reply:
Fixed.